lib/jekyll-github-metadata.rb

```diff
@@ -1,4 +1,5 @@
 require 'octokit'
+require 'digest'
```

Comment:
I'm starting to think each file should have its own requires. They're aggressively cached anyway so if there's any duplication, we should be fine. This would mean we could require a file by itself and it'd pull in all the files it needs.